IP-Protection.md

```diff
@@ -0,0 +1,24 @@
+Chrome's IP Protection is a new privacy feature for Incognito mode designed to limit cross-site tracking by hiding a user's real IP address from specific third-party domains. This protection will be enabled for signed-in Chrome users
```

Comment:
Chrome IP Protection is not yet implemented in any Chrome version yet, can you update the line to reflect that.

Reply:
Updated introduction to reflect the proposed IP protection